README.md

```diff
@@ -69,7 +69,24 @@ This directory is contains any data required for the applications to run (for ex
 
 ## Installation
 
-### Dependencies
+### OpenCV with HPX backend (recommended)
+To build the official OpenCV repository with the HPX backend you will need to install:
+
+1. [HPX](http://stellar-group.org/libraries/hpx/docs/)
+    * Note: Add the following CMake option while building HPX:
+    ```bash
+    -DHPX_WITH_CXX11=ON
+    ```
+2. [OpenCV (4.0.0-alpha or greater)](https://opencv.org/releases/)
+    * Note: The start_stop version of the backend originally conceived has been removed in the official opencv repository.
+    * Add the following parameter to your cmake command (cmake should handle the rest):
+    ```bash
+    -D WITH_HPX=ON
+    ```
+
+### OpenCV with HPX backend (this repository)
+**_IMPORTANT: This is for experimentation only, for the official OpenCV backend read above._** 
```

Comment:
Actually, this was a temporary "work-around" until the PR to OpenCV has been merged. Now this section can be deleted.